consensus/replay.go

```diff
@@ -324,7 +324,7 @@ func (h *Handshaker) ReplayBlocks(
 			}
 
 			if res.ConsensusParams != nil {
-				state.ConsensusParams = types.PB2TM.ConsensusParams(res.ConsensusParams, state.ConsensusParams.Block.TimeIotaMs)
+				state.ConsensusParams = state.ConsensusParams.Update(res.ConsensusParams)
```

Comment:
Interesting, this works the other way now: `state.ConsensusParams.Update` updates everything (or overwrites every filed if necessary)  *besides* blockTimeIotaMs. This comes with the neat side effect that we can delete `pb2tm.ConsensusParams(...)` 👍